tests/integration/suite/daprd/subscriptions/declarative/operator/v1alpha1/bulk/http.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bulk
```

Comment:
Could it use basic func and only change some params, I see many similar codes, maybe we can abstract them.